### PR TITLE
Add implementation of the 'Next' button.

### PR DESCRIPTION
**Pull request by elfing44**
for the Popular novel section  

image

 * Add implementation of the 'Next' button.

* Add page selection.

### CODE REVIEW

File: src/components/accordions/PopularNovels/index.tsx [hunk 1]
```diff
@@ -37,14 +60,31 @@ export default function PopularNovels() {
...
           Next Page
         </Button>
+        <ToggleButtonGroup
+          value={currentIndex}
+          exclusive
+          onChange={(event, value) => fetchNovelsByIndex(value)}
+        >
+          {Array.from({ length: maxIndex }, (_, index) => index + 1).map(
+            number => (
+              <ToggleButton key={number} value={number}>
```
Comment (nyagami) [inline src/components/accordions/PopularNovels/index.tsx:82]: you should use an unique key like `'novel_page_' + index`

**Comment by elfing44** [comment]
Add Fix update of the summary and chapter text.

File: src/components/accordions/PopularNovels/index.tsx [hunk 2]
```diff
@@ -37,14 +60,31 @@ export default function PopularNovels() {
...
           Next Page
         </Button>
+        <ToggleButtonGroup
+          value={currentIndex}
+          exclusive
+          onChange={(event, value) => fetchNovelsByIndex(value)}
+        >
+          {Array.from({ length: maxIndex }, (_, index) => index + 1).map(
+            number => (
+              <ToggleButton key={'novel_index_${number}'} value={number}>
```
Comment (nyagami) [inline src/components/accordions/PopularNovels/index.tsx:82]:
```suggestion
              <ToggleButton key={`novel_page_index_${number}`} value={number}>
```

File: src/components/accordions/PopularNovels/index.tsx [hunk 3]
```diff
@@ -37,14 +60,31 @@ export default function PopularNovels() {
...
+          onChange={(event, value) => fetchNovelsByIndex(value)}
+        >
+          {Array.from({ length: maxIndex }, (_, index) => index + 1).map(
+            number => (
+              <ToggleButton key={`novel_page_index_${number}`} value={number}>
+                {number}
+              </ToggleButton>
+            ),
+          )}
+        </ToggleButtonGroup>
```
Comment (nyagami) [inline src/components/accordions/PopularNovels/index.tsx:87]: I dont think we should use toggle for this case, 
try these steps: 
- go to page 2
- re-click on page 2
- press next page -> web go to page 1.
----
I think only list is enough.

Comment (elfing44) [reply src/components/accordions/PopularNovels/index.tsx:87]: I just tested your scenario but last step I'm coming to page 3 which is well awaited

Comment (nyagami) [reply src/components/accordions/PopularNovels/index.tsx:87]: dont fetch page 3 before you do the last step
step 1:
image
step 2:
image
step 3:
image

Comment (elfing44) [reply src/components/accordions/PopularNovels/index.tsx:87]: What browser are you on? I tried firefox, edge and opera gx. I can't get back to step 2 or the item is deselected.

Comment (nyagami) [reply src/components/accordions/PopularNovels/index.tsx:87]: I'm using chrome

Comment (elfing44) [reply src/components/accordions/PopularNovels/index.tsx:87]: I've just tried on chrome, but I still don't reproduce. 
I may be stupid and I don't understand the step.
You could do a video discord to show me.

Comment (elfing44) [reply src/components/accordions/PopularNovels/index.tsx:87]: I finally managed to reproduce on a different plugin than the one I'm on which I tested.

**Review by nyagami — APPROVED** [review]

**Pull request merged**